config/initializers/devise.rb

```diff
@@ -14,7 +14,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'quepid@o19s.com'
+  config.mailer_sender = ENV['EMAIL_SENDER']
```

Comment:
you followed the pattern of other ENV in this file, however I tend to put everything in `customize_quepid.rb` file as an attribute.